### ISSUE TITLE
Disable street and place name prediction (was Predictions of addr:street unwanted)

### ISSUE THREAD

**Issue by duiveldoder** [MarcusWolschon/osmeditor4android#2188]
this isn't a bug - in my area a lot of streets don't have names or not yet mapped, mostly street names simply don't exist.

I have this preset for
addr:country
addr:province
addr:city
addr:postcode
addr:quarter
addr:neighborhood
addr:housenumber
addr:street
addr:place

Now this all works fine, except the street .. it keeps suggesting me nonsense street names

How can I disable this behavior?

These are the two lines in the xml:

```
<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>	
```

What do I need to add to these lines - to make sure no nonsense is being suggested, because it's kinda annoying to always first delete the entry in the addr:street before I can add the addr:place properly

Note - I do NOT wish to remove the street entry itself - because where a street has a name, or a place has proper street names for the 2+ streets within that place, obviously there I go for street name and skip place.

I really just need to know - how I can about the addr:street=suggestion... get rid of the suggestion.

Thank you

**Comment by simonpoole** [comment]
There is currently no way to suppress vicinity search and suggestions for addr:street and addr:place in the property editor.

**Comment by simonpoole** [comment]
Follow up question to this: in the case where there is "no street/place name" what do the addresses look like? The solution for when there is actually an object with the street/place role is to map it, but the other case might need some custom code or a preference.

**Comment by simonpoole** [comment]
ping @duiveldoder

**Comment by duiveldoder** [comment]
Thank you 🙏🏻

**Comment by simonpoole** [comment]
> Thank you 🙏🏻

An answer to https://github.com/MarcusWolschon/osmeditor4android/issues/2188#issuecomment-1517388462 would be good to determine if this could potentially be handled by more general configuration or if this simply needs a preference.

**Comment by duiveldoder** [comment]
> Follow up question to this: in the case where there is "no street/place name" what do the addresses look like (and where, roughly, is this)? The solution for when there is actually an object with the street/place role is to map it, but the other case might need some custom code or a preference.

There's always the option for place.

Thing is, I just like the possibility to turn off the prediction.

Then no matter what the issue is, I  have an empty value and can type in the right value manually.

If then the feature "use previously entered value for this key" exists I can quickly add data without erroneous predictions.

Where this is, Malaysia and Philippines.. Highly undermapped areas, both rural and cities.

And quite often false names for street which someone copied from Google it seems, and totally wrong.

Because so often there's so much wrong, it's better if I can enable prediction where already mapped properly and disabled where not.

**Comment by simonpoole** [comment]
Just to explain: currently we search in a hardwired 200m radius for street and place names. One of the ways of disabling the prediction would be to simply make the radius configurable. So you you could set it say to 10 metres and if there was no street or place with a name in the radius it would predict nothing, if there was, then you would get the name.

> And quite often false names for street which someone copied from Google it seems, and totally wrong.

Specific note: in this case the solution is to map the correct name and then reset address prediction in Vespucci.

**Comment by duiveldoder** [comment]
The configurable radius is fine by me.

The map first, is a bit hard when I'm field surveying an area for addresses..

Quite often there's no internet where I could upload that, mapping here is quite often offline due to bad network coverage.

That's why I have osm tiles, sat imagery all preloaded from desktop apps and then manually copied to the phone.

And then I do at home down loadings the data from OSM before I go out.

**Comment by simonpoole** [comment]
@duiveldoder btw just wrt to editing offline see vespucci.io/tutorials/offline/ and https://vespucci.io/tutorials/custom_imagery_mbtiles/

**Comment by duiveldoder** [comment]
> I just realized that I don't believe that I completely understand your workflow, are you using Vespuccis address prediction, or are you applying an address preset and then entering things by hand?

address preset and then by hand.

**Comment by simonpoole** [comment]
@duiveldoder could you attach your preset here? 

If you are not using the address prediction function, then what is likely happening is that the preset you are using has the `use_last_as_default="force"` for `addr:street` and  `addr:place` set which will automatically add the last used values for those fields. If you don't have that set I don't quite see how you are automatically getting values added.

See also the note with respect to preset ordering here http://vespucci.io/help/en/Presets/#managing-presets

**Comment by duiveldoder** [comment]
`<?xml version="1.0" encoding="UTF-8"?>
<presets xmlns="http://josm.openstreetmap.de/tagging-preset-1.0" version="1.0" shortdescription="Baguio Addresses" description="Adding Addresses to Baguio City - organized per Barangay and Purok">
	<group name="Camdas Subdivision">
		<item name="Purok 01" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 1" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 02" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 2" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 03" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 3" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 04" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 4" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 05" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 5" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 06" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 6" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 07" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 7" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 08" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 8" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 09" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 9" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
		<item name="Purok 10" type="node,closedway">
			<label text="Addresses Form" />
			<label text="..... Mandatory ....." />
			<text key="addr:housenumber" text="House number" default="" delete_if_empty="true"/>
			<text key="addr:street" text="Street name" default="" delete_if_empty="true"/>
			<text key="addr:place" text="Subdivision/Village/etc" default="" delete_if_empty="true"/>
			<combo key="building" text="Building" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<combo key="building:use" text="Building:use" default="house" values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,government,college,school" display_values="residential,apartments,house,detached,semidetached_house,cabin,kiosk,shed,roof,garage,commercial,hotel,retail,office,industrial,warehouse,church,chapel,shrine,public,Barangay Hall/Outpost,college,school" />
			<text key="name" text="Name of Building" default="" delete_if_empty="true" />
			<combo key="disused" text="Disused? Minor repairs." default="" values="yes" delete_if_empty="true" />
			<combo key="abandoned" text="Abandoned? Major repairs." default="" values="yes" delete_if_empty="true" />
			<text key="note" text="Note" default="" delete_if_empty="true"/>
			<key key="source" value="survey;maxar" />
			<key key="addr:province" value="Benguet" />
			<key key="addr:city" value="Baguio"  />
			<key key="addr:postcode" value="2600"  />
			<key key="addr:country" value="PH"  />
			<key key="addr:neighbourhood" value="Purok 10" />
			<key key="addr:quarter" value="Camdas" />
			<key key="fixme" value="address" />
			<text key="check_date" text="Automatic check_date generation" javascript="df = new java.text.SimpleDateFormat('yyyy-MM-dd');df.format(new java.util.Date());" />
		</item>
	</group>
</presets>
`

**Comment by simonpoole** [comment]
If I apply the preset this is what happens (no automatic street name):

[Purok 01.webm](https://github.com/MarcusWolschon/osmeditor4android/assets/1011860/ffe4f891-ee80-4cc2-a9f7-85b41ac2bd6b)

**Comment by simonpoole** [comment]
@duiveldoder I would like to resolve this. As I pointed out above, I don't get place or street name automatically filled in when I apply your preset, and that would seem to be the behaviour you desire? Or do you just don't want any suggestions at all, even if they just appear in the drop down list?

**Comment by duiveldoder** [comment]
I'll try it out again next week where there are no street names on location.

As I think recently when I tried it again with my new phone with fresh install it worked as I expected.

But let me try again next week I'm again on site in areas with no names so then I can do another real field test

**Comment by simonpoole** [comment]
There is actually one situation in which a predicted street/place name will be automatically inserted: if you manually add addr:street or addr:place in the "Details" tab. It seems unlikely that this is what you were running in to though.

**Comment by simonpoole** [comment]
ping @duiveldoder

**Comment by simonpoole** [comment]
Stale
